transcendence-app/src/shared/local-file/infrastructure/db/local-file.entity.ts

```diff
@@ -9,7 +7,7 @@ export enum LocalFileKeys {
   CREATED_AT = '"createdAt"',
 }
 
-export class LocalFileEntity implements BaseEntity {
+export class LocalFileEntity {
```

Comment:
Why do we have to give up on this particular inheritance hierarchy?

Reply:
The `BaseEntity` had one optional field(`id`), which the `AuthProviderEntity` doesn't have. TypeScript was throwing some errors, and the easiest way to fix them was to remove the `BaseEntity` interface.